services/comm/app/operations/message_create.rb

```diff
@@ -94,7 +94,7 @@ def setup_message(ctx, params:, **)
     # TODO: If the tenant has a provider set, use the tenant's provider
     # else default to platform default provider
     ctx[:model] = Message.new(params)
-    ctx[:model].provider_id = Providers::Aws.first.id
+    ctx[:model].provider_id = Providers::Aws.first&.id
```

Comment:
This is no longer needed. Provider for messages is optional now and if we need to specify `provider_id` for the specific message we should pass it in `params`.

Let's remove this line and make sure that tests are passing.